src/main/java/com/ibm/cloud/sdk/core/service/BaseService.java

```diff
@@ -103,9 +103,6 @@ public BaseService(final String name, Authenticator authenticator) {
 
     // Configure a default client instance.
     this.client = configureHttpClient();
-
-    // temp: set any external configuration from the constructor
-    configureService(name);
```

Comment:
What are the implications of removing this call here?  Won't this result in different, possibly incompatible results for existing users?

Reply:
Well this idea is if the user specifies ``includeExternalConfig`` to be true, then the constructor in the generated code will make this call to ``configureService`` and not the constructor in the BaseService. However, if a user uses a version of the generator that doesn't have that change, and uses this new version of the core, then yes it would be a breaking change. I think the idea was to be specific about that in the compatibility chart.

Reply:
@mkistler - though I'm still trying to grasp on how you guys do version releases. I think before, if a user didn't specify ``includeExternalConfig`` to be true, the code that looks for external config was still being ran, but it wouldn't actually configure the service. Is that correct?

Reply:
Yes, this will be a change that will affect users.   This new version of the java core will need to be versioned as a breaking change, then when the new version of the generator is tagged (2.2.0?) we'll need to update the compatibility chart accordingly.
For Watson services (i.e. includeExternalConfig is enabled), it won't constitute a breaking change because the new version of the generator will generate their ctors such that they call configureService().   
Only non-Watson services will be affected, but only if applications try to use the new java core with old generated code.  Clear as mud, right?

We also have another related issue that would be good for all three of us to talk about when our schedules line up.